python/usb_console/radio_session.py

```diff
@@ -36,8 +41,16 @@ def __init__(self, device_name, datastore, logger):
         self.datastore = datastore
         self.logger = logger
 
-        # Simulation
-        self.overriden_variables = set()
+        #email        
+        with open(os.path.join(os.path.dirname(os.path.abspath(__file__)), 'configs/confidential.json')) as config_file:
```

Comment:
Can you rename confidential.json to something like "radio_keys.json"? Makes the file more descriptive.